.github/workflows/codeql-analysis.yaml

```diff
@@ -10,10 +10,10 @@ name: Code Scanning - Action
     - cron: '44 11 * * 3'
 
 jobs:
-  CodeQL-Build:
-    # CodeQL runs on ubuntu-latest, windows-latest, and macos-latest
-    runs-on: ubuntu-latest
-
+  analyze:
+    name: Analyze
+    runs-on: ${{ (matrix.language == 'swift' && 'macos-latest') || 'ubuntu-latest' }}
```

Comment:
since we don't have swift, i guess this is just template from somewhere?

Reply:
yes, generated by GitHub we only use Ubuntu, so it can be removed 👍🏾